visidata/path.py

```diff
@@ -74,7 +74,10 @@ def __getattr__(self, k):
         if hasattr(self.__dict__, k):
             r = getattr(self.__dict__, k)
         else:
-            r = getattr(self._path, k)
+            if self.__dict__.get('_path', None) is not None:
+                r = getattr(self._path, k)
+            else:
+                raise AttributeError
```

Comment:
I think this should be `raise AttributeError(k)`, no?

Reply:
Great idea!